### PR TITLE
Release v3.1

### PR DESCRIPTION
**Pull request by inodb**
Add new data:

- Level 1 & 2 10X Visium data from WUSTL PDAC and CCRCC studies
- Level 2 & 3 single cell and single nucleus RNA-sequencing data from WUSTL PDAC study
- Additional Level 1-3 single cell RNA-sequencing files from WUSTL PDAC study
- Level 1-4 MSK single cell RNA-seq data

Note also: One Vanderbilt sample was removed as it was not used in HTAN studies

From:
Screen Shot 2023-03-10 at 7 00 18 PM
To:
Screen Shot 2023-03-10 at 7 00 21 PM

### CODE REVIEW

File: pages/static/data-updates.html [hunk 1]
```diff
@@ -4,6 +4,20 @@
...
+<h2 id="2023-03-10">March 10th, 2023</h2>
+<p>New data is available from WUSTL and MSK atlases as part of Release 3.1:</p>
+<ul>
+    <li>Level 1 & 2 10X Visium data from WUSTL PDAC and CCRCC studies</li>
+    <li>
+        Level 2 & 3 single cell and single nucleus RNA-sequencing data from
+        WUSTL PDAC study
+    </li>
+    <li>
+        Additional Level 1 single cell RNA-sequencing files from WUSTL PDAC
```
Comment (clarisse-lau) [inline pages/static/data-updates.html:16]: Change to: 
"Additional Level 1-3 single cell RNA-sequencing files from WUSTL PDAC study"

WUSTL added a few higher level ScRNA-seq files for the `HTA12_6_13` sample on Wednesday, so updating to reflect that

Comment (inodb) [reply pages/static/data-updates.html:16]: In that case, maybe I combine the two lines then?

> Level 1- 3 single cell and single nucleus RNA-sequencing data from WUSTL PDAC study

Comment (clarisse-lau) [reply pages/static/data-updates.html]: Thanks, lemme double check with Vesteinn really quickly. 
The intention was to distinguish between their 2 PDAC submissions, may re-work it a little bit

Comment (clarisse-lau) [reply pages/static/data-updates.html]: Ok, let's leave it as is :) Thank you!

**Pull request merged**